code/style.css

```diff
@@ -1,4 +1,175 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* Header */
+.header {
+  display: flex;
+  margin: 20px;
+  width: auto;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.logo {
+  width: 100px;
+}
+
+h1 {
+  font-family: 'Permanent Marker', cursive;
+  font-size: 50px;
+  color: #B677D4;
+  margin: 15px;
+}
+
+.navigation {
+  display: flex;
+  align-items: flex-end;
+  margin: 20px;
+}
+
+/* Main content */
+.main-container {
+display: flex;
+flex-direction: column;
+}
+
+/* Big news */
+.big-news {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: auto;
+  height: auto;
+}
+
+.sushi {
+  width: 100%;
+  height: auto;
+  margin: 0;
+  border: 1px solid;
+  padding: 6px;
+  box-shadow: 2px 8px;
+
+}
+
+h2, h3 {
+  font-family: 'Permanent Marker';
+  padding: 0%;
+  margin: 2px;
+}
+
+.text-big-news {
+  width: auto;
+  height: auto;
+  margin: 0 30px;
+  font-family: 'Roboto';
+  text-align: justify;
+}
+
+/* Line */
+hr {
+  display: block;
+  border: 2px solid #81F0A3;
+  width: auto;
+  margin: 20px;
+}
+
+/* Other news */
+.other-news {
+  display: grid;
+  grid-gap: 5px;
+  grid-template-columns: 1fr;
+  height: auto;
+  width: 100%;
+}
+
+.small-news {
+  height: auto;
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-self: center;
+  margin: 15px;
+  background-color: white;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
I like the look of your box shadows, not too harsh but very subtle. Need to try this out